bitnami/kube-state-metrics/README.md

```diff
@@ -50,95 +50,96 @@ The command removes all the Kubernetes components associated with the chart and
 
 The following table lists the configurable parameters of the kube-state-metrics chart and their default values.
 
-| Parameter                                  | Description                                                                                                   | Default                                                    |
```

Comment:
Could you try to preserve the old format? It is easier to review the changes.

Reply:
Unfortunately the columns will not be aligned with this new addition